### DISCUSSION TITLE
Update to 3.2.2

### DISCUSSION THREAD

**Discussion by Choppers67** [garrynewman/Control4.Jailbreak#11] [Q&A]
So if I update my hardware to version 3.2.2 and composer is still 3.2.1  will the Jailbreak still work??

**Comment by cgbale** [comment]
Update Composer to 3.2.2, then apply the Jailbreak certificates, then connect to and update your controller to 3.2.2. It may take up to an hour for the controller to update.

**Reply by Choppers67** [reply]
Worked like a charm! Yes the Composer and the Director have to be the same version to work. I updated the the director first then Composer. Not sure if possible to do it the other way but worked fine.

**Reply by gdamery** [reply]
The opposite seems to work too.   I had been using Composer HE 3.2.2.  So I downloaded Composer Pro 3.2.2 and ran the patches, and it worked.  A miracle!

**Reply by Choppers67** [reply]
Thanks for the feedback!  Glad that automation enthusiasts like us can still work under the hood and do the mods we want. I hope Gary Newman keeps this going and a big shout out to him!
